make/gen.cmake

```diff
@@ -19,7 +19,7 @@ function(V2X)
   # files.  DO_NOT_APPLY_VERILOG_IMAGE_GEN suppress this default.
   set(options DO_NOT_APPLY_VERILOG_IMAGE_GEN)
   set(oneValueArgs NAME TOP_MODULE)
-  set(multiValueArgs SRCS)
+  set(multiValueArgs SRCS GENERATEDXMLDEPS GENERATEDVERILOGDEPS)
```

Comment:
Why should the XML dependencies need to input?  And why not just have the verilog dependencies communicated via SRCS INCLUDE files?

Reply:
as for Verilog:
SRCS are parsed as N template. The intention here is that the VERILOG deps were already generated and should not be processed furher.

XML deps:
some generated xmls are included by other generated xmls. This is to ensure that those will be included in the deps list.

Reply:
This should be handle via the INCLUDE_FILES property, otherwise the dependencies will not propagate to other tools.  I'm not clear what these arguments do that could not have been done with the INCLUDE_FILES property and append_file_dependency